src/app/pages/MainPage/MainPage.module.css

```diff
@@ -0,0 +1,23 @@
+.body_div {
```

Comment:
same here - body (for me) is the overall setting - this might confuse people (me )  ;)